ts/parts/Axis.ts

```diff
@@ -7401,7 +7406,7 @@ extend(Axis.prototype, /** @lends Highcharts.Axis.prototype */{
 
             // Draw the cross
             if (!graphic) {
-                this.cross = graphic = this.chart.renderer
+                this.cross = graphic = chart.renderer
```

Comment:
Nitpicking again: since you change `this` -> `chart`, then `this.cross` should be changed too? ;)

Reply:
Here `this` is referring to the Axis. Do you mean creating `axis` variable for `this`?

Reply:
Ah, my bad, thanks!